src/SourceBuild/content/repo-projects/installer.proj

```diff
@@ -29,7 +29,7 @@
     <BuildArgs Condition="'$(TargetOS)' != 'windows'">$(BuildArgs) /p:CoreSetupRid=$(TargetRid)</BuildArgs>
 
     <!-- Consume the source-built Core-Setup and toolset. This line must be removed to source-build CLI without source-building Core-Setup first. -->
-    <BuildArgs>$(BuildArgs) /p:PublicBaseURL=file:%2F%2F$(SourceBuiltAssetsDir)</BuildArgs>
+    <BuildArgs>$(BuildArgs) /p:PublicBaseURL=file:%2F%2F$(ArtifactsShippingPackagesDir)</BuildArgs>
```

Comment:
This line worries me a bit. I'm not sure if installer needs the transport feed as well. If it does, how would we pass that in? cc @mmitche

Reply:
OK the build failure is telling that we really need both local feeds.

Reply:
It will need the transport feed dir too. We don't typically separate blob assets by shipping and non-shipping. Those go to a single storage locaiton. Packages go to two locations.

The reason being is mainly that we can install SDK +, add shipping feed Y, and test.